cleanlab_studio/cli/api_service.py

```diff
@@ -103,7 +108,11 @@ async def upload_rows_async(
         ]
 
         filepath_to_post = get_presigned_posts(
-            api_key=api_key, dataset_id=dataset_id, filepaths=filepaths, media_type=modality.value
+            api_key=api_key,
+            dataset_id=dataset_id,
+            filepaths=filepaths,
```

Comment:
In V2, we still use the user-supplied filepath for the file extension. Object keys are composed of `<prefix>/hash(<row_id>).<extension>`

From what I can tell, the suffix is not necessary for pillow to read the images, so we could remove this if we want. I kept it in case we ever did have logic that's dependent on the file extension (though we also have the `labels.csv` as a source for that).

Reply:
Good point, let's keep it for now. Even if the extension is not necessary, it's probably convenient to keep around.